prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -57,7 +57,9 @@ public async Task<IEnumerable<OrganizationSearchViewModel>> GetOrganizationsAsyn
                 .If(!string.IsNullOrWhiteSpace(searchOptions.TextSearch), q => q
                     .Search(
                         o => o.Name,
-                        o => o.DisplayId.ToString())
+                        o => o.DisplayId.ToString(),
+                        o => o.SigningAuthority.FirstName,
+                        o => o.SigningAuthority.LastName)
```

Comment:
Cas also add 
``` o => o.SigningAuthority.FirstName + " " + o.SigningAuthority.LastName) ```
To allow for more refined search on whole name so that they will most certainly filter to what they want to see.

Reply:
Hmm let me try that, I tried the otherway with "${o.SigningAuthority.FirstName} ${o.SigningAuthority.FirstName}" and code didn't like that